src/psyclone/psyir/transformations/arrayrange2loop_trans.py

```diff
@@ -78,127 +78,6 @@ class ArrayRange2LoopTrans(Transformation):
 
     '''
```

Comment:
In line 44 the import of symbolic maths can be removed.

Reply:
Done